README.md

```diff
@@ -118,6 +118,16 @@ The instructions below assume you're on Linux.
      pip install torchcodec
      ```
 
+## Benchmark results
+
+The following results were obtained by running the [benchmark_decoders.ipynb](./benchmarks/decoders/benchmark_decoders.ipynb) on a lightly-loaded 22-core machine. We first get the operation latency for various seek and decode patterns in a loop from a single python thread
```

Comment:
Our README and other files restrict the line length to ~80, we should try to keep it this way (it also makes review easier to locate comments on specific lines).